tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 360x360 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#let's do some posterior predictive checks to see if the\n",
+    "#observation `x_o` remains within the support of the posterior\n",
+    "from sbi.analysis import pairplot\n",
+    "\n",
+    "fig, ax = pairplot(\n",
+    "    samples=posterior_data,\n",
+    "    points=x_o,\n",
+    "    limits=list(zip(.75*x_o.flatten(), 1.25*x_o.flatten())),\n",
+    "    upper=['kde'],\n",
+    "    diag=['kde'],\n",
+    "    figsize=(5,5),\n",
+    "    labels=[rf\"$\\theta_{d}$\" for d in range(3)]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "The posterior is well aligned with the observation. On top, the observation falls into the support of the posterior samples. That is indicative that our posterior estimate matches the true posterior very well."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### Running SBC\n",
+    "\n",
+    "We have a working and trained posterior at this point! Hurray! Let's look at the SBC metrics now."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "num_sbc_runs = 1_000 #the higher sbc runs the better\n",
```

Comment:
technical note: for very large `N` any KS test will eventually become significant when testing uniformity. Not sure whether we want to mention that though. Maybe change the comment to a concrete recommendation for how many samples to use, e.g., 100s or ideally 1000.

Reply:
if and only if, people keep tying themselves to the KS_alpha=0.05 confidence level bound.
813e66b